conf/global.properties

```diff
@@ -143,6 +144,9 @@ postfixrequests.list=mail:SendMsgRequest,\
                                         mail:BounceMsgRequest,\
                                         mail:ForwardAppointmentInviteRequest,\
 										admin:ContactBackupRequest
+ewsrequests.list=ewsmsg:SyncFolderItems
+ewsrequests.delay = 4000
+EwsDelay.check = true
```

Comment:
This should be false by default. As we don't want this delay in legacy

Reply:
done